api/queries_repo.go

```diff
@@ -61,6 +62,11 @@ func (r Repository) RepoHost() string {
 	return r.hostname
 }
 
+// RepoHasWiki checks if a wiki exists in a repo
+func (r Repository) RepoHasWiki() bool {
```

Comment:
This function would be better named as just `HasWiki`, since the "Repo" prefix is redundant on a Repository struct. (Other functions named `RepoOwner/Name/Host` have the "Repo" prefix only because they are part of an interface that is shared between possibly non-repo objects.)

However, I don't think this accessor provides any value, so you can just remove the function and have the caller inspect the `HasWikiEnabled` field directly.